### ISSUE TITLE
Support hold a participant through update method

### ISSUE THREAD

**Issue by brayann** [twilio/twilio-php#368]
I'm trying to do the following, it gives no error, but also doesn't put the participant on hold.

```
$this->client
	->conferences($conferenceSid)
	->participants($memberCallSid)
	->update(['Hold' => 'true']);
```

On the [docs](https://www.twilio.com/docs/api/rest/participant#instance-post-example-1) it gives an example of holding a participant only by using curl. Is this not supported on the library, or am I doing something wrong?

Thanks,

**Comment by ihumanable** [comment]
This looks to be an error in our definitions for that endpoint, we are looking into a solution for this issue, thank you for bringing it to our attention

**Comment by brayann** [comment]
Sure, I'd be happy to help with testing or even fixing it, lmk.

**Comment by ihumanable** [comment]
This is fixed in 5.2.0 release
